airbyte-config/init/src/main/resources/seed/source_definitions.yaml

```diff
@@ -299,6 +299,14 @@
   documentationUrl: https://docs.airbyte.com/integrations/sources/faker
   sourceType: api
   releaseStage: alpha
+- name: Fauna
+  sourceDefinitionId: 3825db3e-c94b-42ac-bd53-b5a9507ace2b
+  dockerRepository: airbyte/source-fauna
+  dockerImageTag: dev
+  documentationUrl: https://docs.airbyte.com/integrations/sources/fauna
+  icon: fauna.svg
+  sourceType: database
+  releaseStage: alpha
```

Comment:
@marcosmarxm I don't think a "dev" version exists. This should be `v0.1.2`.

https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-fauna/Dockerfile#L37 should be changed to be a real version number

Reply:
@evantahler my bad! I think latest changes from https://github.com/airbytehq/airbyte/pull/19534 fix it! thanks for catching this.